### ISSUE TITLE
Clicking on a Plantmaster force-disconnects with enough seeds inside

### ISSUE THREAD

**Issue by du5tball** [goonstation/goonstation#10595]
### Labels

[BUG]

### Description

When there's a lot of seeds in the Plantmaster, clicking it force-disconnects the player

### Steps to reproduce

1. Load a lot of seeds in the plantmaster (75, maybe more)
2. Click plantmaster
3. no profit

### Expected Behavior

Either the plantmaster doesn't allow more than X (maybe 50?) seeds, or it handles it better

**Comment by frawhst** [comment]
I sure fucking hope this is the bug because I’ve known about this for months and have 0 reproduction steps

**Comment by du5tball** [comment]
If it helps: I've tried making Lifeweed via infusions in there, ran a few hydro basins and dumped pretty much all the seeds in there. After I dumped the 75+, I got a few back out, closed it, opened it again and then it crapped out. There was also an empty beaker in there.
I can at least say it put one other person into the "stares blankly" state, so it wasn't something local. If the serverlogs help: It happened 5 minutes before I posted this, on goon2.
